### ISSUE TITLE
Strange metrics?

### ISSUE THREAD

**Issue by Mercury13** [notofonts/egyptian-hieroglyphs#16]
As U15 is ready, let’s sort out another problem: bad metrics.

Most squares have Y coordinate −260…967.
![image](https://github.com/notofonts/egyptian-hieroglyphs/assets/10164882/82ad0ed9-76cd-40c1-bac9-5138ec17f31e)

Most people have it 0…1001.
![image](https://github.com/notofonts/egyptian-hieroglyphs/assets/10164882/ae81fdf8-97f6-4fc1-bb20-aebd52fc0fb8)

How to sort it out better?

**Comment by simoncozens** [comment]
Yeah, this is bad and hurts us when trying to do WASM-based quadrat packing. I think the ideal would be for everything to be more-or-less 0...1000.

Maybe it's time to talk my old friend Roland, who's now an Egyptology professor somewhere.

**Comment by Mercury13** [comment]
@simoncozens Please talk to Roland, and we’ll think what to do.

**Comment by simoncozens** [comment]
Is this the same as #7?

**Comment by Mercury13** [comment]
I don’t even know. Wherever we discuss, we should figure out what to do, and I’ll write the program to transform it.
